app/views/conference/_event.xml.haml

```diff
@@ -0,0 +1,20 @@
+- event.public?
+  %event{ :guid => event.guid, :id => event.id }
+    %date=event.start_time.iso8601
+    %start=event.start_time.strftime("%H:%M")
+    %duration=event.event_type.length_timestamp
+    %room=event.room.name
+    %type=event.event_type.name
+    %language="en" ## fix me, add language property to Event/Conference?
+    %slug=event.slugify
+    %title=event.title
+    %subtitle=event.subtitle
+    %track=event.track.name
+    %abstract=event.abstract
+    %recording
```

Comment:
Currently only license (could inherit from the project, but sometimes it makes sense that individual talks have different licenses) and opt-out (i.e. when the speaker does not want to be recorded).